infra/main.tf

```diff
@@ -0,0 +1,106 @@
+terraform {
+    required_providers {
+        google = {
+            source = "hashicorp/google"
+            version = "4.37.0"
+        }
+    }
+}
+
+variable "vm_name" {
+    type = string
+}
+
+variable "user" {
+    type = string
+}
+
+variable "credentials_file" {
+    type = string
+}
+
+provider "google" {
+    credentials = file("${var.credentials_file}")
+
+    project = "textile-310716"
+    region  = "us-west1"
+    zone    = "us-west1-b"
+}
+
+resource "google_compute_instance" "validator" {
+    name         = "${var.vm_name}"
+    machine_type = "e2-standard-2"
+    tags         = ["https-server", "grafana", "ssh-vm"]
+    deletion_protection = false
+
+    boot_disk {
+        initialize_params {
+            image = "ubuntu-2204-jammy-v20220902"
+            type = "pd-ssd"
+            size = "30"
+        }
+    }
+    
+    network_interface {
+        network = "default"
+        access_config {}
+    }
```

Comment:
Prob out of scope for Terraform, but we should remember (maybe comment here?) that the operator should set up firewall rules, but perhaps that's somewhat obvious.

Reply:
We can set the firewall rules here or add a `depends on` so Terraform checks if firewall rule exists